config/default.yml

```diff
@@ -1505,7 +1505,9 @@ Lint/EmptyBlock:
   Description: 'This cop checks for blocks without a body.'
   Enabled: pending
   VersionAdded: '1.1'
+  VersionChanged: '1.3'
   AllowComments: true
+  AllowEmptyLambdas: true
```

Comment:
Can you make the configuration name consistent with either `AllowEmptyLambda` or `AllowEmptyLambdas`?

Reply:
🤦 fixing!

Reply:
Fixed now, sorry about that.